dashboard/test/ui/support/hooks.rb

```diff
@@ -11,3 +11,11 @@
     saw_failure = true
   end
 end
+
+Before('@as_student') do
+  steps 'Given I manually sign in as "TestStudent"'
```

Comment:
I think we want to sleep for a couple seconds here. There's a race condition with signing in and the sign-in button that sleeping might avoid. @philbogle may have more details